### PR TITLE
Deathstats

### PR DESCRIPTION
**Pull request by jroinc**
## About The Pull Request

Adds stat tracking for human deaths and revives.

## Why It's Good For The Game

More accurate data before I overnerf revives to see if it's actually needed first.

Additional statistics!

## Changelog
:cl:
expansion: Human deaths and defib-based revives now are logged as a statistic.
/:cl:

### CODE REVIEW

**Comment by BraveMole** [comment]
ok but you're not tallying deaths, so it will stay at zero

**Comment by jroinc** [comment]
> 
> 
> ok but you're not tallying deaths, so it will stay at zero

What'd I fuck up? Deaths should have already been tallied in an earlier PR, it's just not displayed.

**Review by SplinterGP — APPROVED** [review]

File: code/datums/gamemodes/game_mode.dm [hunk 1]
```diff
@@ -420,6 +420,8 @@ Sensors indicate [numXenosShip || "no"] unknown lifeform signature[numXenosShip
 		dat += "[GLOB.round_statistics.grenades_thrown] total grenades exploding."
 	else
 		dat += "No grenades exploded."
+	if(GLOB.round_statistics.total_human_deaths)
+		dat += "[GLOB.round_statistics.total_human_deaths] people were killed, among which [GLOB.round_statistics.total_human_revives] were revived. For a [(GLOB.round_statistics.total_human_revives / max(GLOB.round_statistics.total_human_deaths, 1)) * 100]% medical revival rate."
```
Comment (SplinterGP) [inline code/datums/gamemodes/game_mode.dm:424]: missing human revives

**Review by SplinterGP — CHANGES_REQUESTED** [review]

File: code/datums/gamemodes/game_mode.dm (hunk 1 above)
Comment (jroinc) [reply code/datums/gamemodes/game_mode.dm:424]: I added the var to round_statistics and it increments on defib revive. What am I not seeing?

**Review by SplinterGP — APPROVED** [review]
nvm

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**